app/controllers/default_routing_controller.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2008-2017, Puzzle ITC GmbH. This file is part of
+#  Cryptopus and licensed under the Affero General Public License version 3 or later.
+#  See the COPYING file at the top-level directory or at
+#  https://github.com/puzzle/cryptopus.
+
+class DefaultRoutingController
+
+  def redirect
```

Comment:
I think it's completely save to just skip authorization here completely. 
make sure you're testing all cases in the new integration test.
logged in and not logged in user for example

Reply:
if the user is not logged in, he's going to be redirected to the login form. we also remember what URL the user tried to access and redirect him after successful login. 
check if this is also possible with the legacy url